manager/integration/tests/test_settings.py

```diff
@@ -113,8 +126,11 @@ def test_setting_toleration():
                       value="key1=value1:NoSchedule; key2:InvalidEffect")
     assert 'invalid effect' in str(e.value)
 
+    common.create_backing_image_with_matching_url(
+        client, common.BACKING_IMAGE_NAME, common.BACKING_IMAGE_QCOW2_URL)
     volume_name = "test-toleration-vol"  # NOQA
-    volume = create_and_check_volume(client, volume_name)
+    volume = create_and_check_volume(client, volume_name,
+                                     backing_image=common.BACKING_IMAGE_NAME)
```

Comment:
It's better to use a separate case to test the backing image. And checking the data is required.
You can refer to the tests in `test_backing_image.py`

Same below

Reply:
Separated case to `test_setting_toleration_with_backing_image`.